content/docs/buildpack-author-guide/create-buildpack/adding-bill-of-materials.md

```diff
@@ -161,10 +163,6 @@ else
     echo "---> Installing gems"
     mkdir -p "$bundlerlayer"
     cat > "$layersdir/bundler.toml" << EOL
```

Comment:
Here it should append IMO, otherwise, it will overwrite the previously set `[types]` table.